### PR TITLE
Add Hebrew translation

### PR DESCRIPTION
**Pull request by omeritzics**
IMPORTANT: While translating, I saw writing mistakes of capital letters (for example: ubuntu instead of Ubuntu, windows instead of Windows) and double spacing (in description-lubuntu), and "some distribution were" instead "some distributions were" (in additional-info-hardware); Please replace "OpenSuse" with "openSUSE", as this is the official way to write the distribution name.

### CODE REVIEW

**Comment by cmllr** [comment]
Hey, thank you for your interest in contributing to this project. Is this an ongoing work (as I see some values still missing)? Can I help you with something? Feel free to ping me :)

**Comment by omeritzics** [comment]
> Hey, thank you for your interest in contributing to this project. Is this an ongoing work (as I see some values still missing)? Can I help you with something? Feel free to ping me :)

Yes, and I am willing to translate more string.
There are original strings in German, should them be translated?

**Comment by omeritzics** [comment]
The direct translation of "How many configuration variables should be filled with default values?" is not understandable.
Is translation of "How much things you want to be pre-configured?" acceptable?

**Comment by omeritzics** [comment]
@cmllr There is no Hebrew word for accurate translation for "imprint". What alternative word do you think will fit the translation?

**Comment by omeritzics** [comment]
Almost finished to translate. @cmllr What alternative translations for "matrix" would you suggest?

File: backend/locale/he.po [hunk 1]
```diff
@@ -0,0 +1,821 @@
...
+msgstr ""
+"קוד המקור מוגן ברישיון שכתוב בקובץ \"LICENSE\", אשר מאוחסן בתוך המאגר אליו"
+" ניתן הקישור."
+
+msgid "about-licenses-database"
+msgstr "מסד הנתונים"
+
+msgid "about-licenses-database-text"
+msgstr ""
+"The underlying data, including the question/answer structure, the result"
```
Comment (cmllr) [inline backend/locale/he.po:171]: If your comment was telated to this translation, you can also aim to translate for following sentence:

"The underlying data, including the question/answer structure, the result"
" information and statistics, are proprietary. You may use them only with our prior"
" permission. The description texts and logos of the various distributions are"
" subject to the licences of the respective copyright holders."

**Comment by cmllr** [comment]
Hey @omeritzics Thank you for continuing work on your translation.

If your remark was related to https://github.com/distrochooser/distrochooser/pull/120/files#r659337840, I would suggest to subsitute the word matrix quite generally with "information". If you don't feel confortable with that, we can also keep that translation open. In some of the other translations similar situations occured and we sticked to the english translation.

**Comment by omeritzics** [comment]
> * On the navigation elements on the left, how is the expected from a RTL reader perspective? Icons on the left of the text or on the right (as on the screenshot, ignore the odd margin for a moment) + same question for the check boxes/ radio boxes on answers
> * The primary buttons (skip, next) in the question area, should they be located as on the screenshot or is the natural reading way that this elements are expected to be on the right (as on the english version, basically)

Hi, thank you for adding RTL!!
The icons should be on the right of the text, and almost everything is in its right place.

On the top-right mergin, the "A project by chmr.eu" text is displayed as "A project by              [GitHub logo] chmr.eu".
Additionally, the links on the top-right mergin should be on the left instead of the right (but this is not a big problem).

**Comment by cmllr** [comment]
@omeritzics I just merged the translation into the master. I want again to thank you for your contribution and your patience!

If you find anything I can still adapt in term of page layout and/ or language semantics (such as icons), feel free to open a new issue.

I added you in the project readme https://github.com/distrochooser/distrochooser#readme 

The update just went live :)

**Comment by omeritzics** [comment]
> 

Thank you too :)

**Pull request merged**